App/routes/operations.py

```diff
@@ -0,0 +1,26 @@
+from flask import jsonify
+from main import app
+
+@app.route("/api/calc/sumar/<num1>/<num2>")
```

Comment:
Please add comments to describe what this function does (docs)

Reply:
Done, Comments added to the function